sdk/src/nonce_account.rs

```diff
@@ -21,13 +24,13 @@ pub fn create_account(lamports: u64) -> RefCell<AccountSharedData> {
 }
 
 // TODO: Consider changing argument from Hash to DurableNonce.
```

Comment:
this `// TODO: ...` was not a good idea. Since you are changing this area, maybe remove this as well.